src/test/java/io/redis/examples/CmdsSortedSetExample.java

```diff
@@ -0,0 +1,482 @@
+// EXAMPLE: cmds_sorted_set
+// REMOVE_START
+package io.redis.examples;
+
+import org.junit.Assert;
+import org.junit.Test;
+import static org.junit.Assert.*;
```

Comment:
It makes sense only to remove lines like this, which are needed only for internal testing. Other imports like `import java.util.HashMap;` should be presented to the end user so they can copy-paste it and run without fixing any imports.

Reply:
Fixed.